tests/wagtail_live/receivers/telegram/test_telegramwebhookreceiver.py

```diff
@@ -0,0 +1,540 @@
+    message = telegram_bot_other_command["message"]
+
+    assert telegram_receiver.handle_bot_command(message=message) is None
+    requests.get.assert_not_called()
+
+
+def test_handle_bot_command_error(
+    telegram_receiver, telegram_bot_command, mocker, caplog
+):
+    json_data = {"ok": False, "description": "Failed sending message"}
+    mocker.patch.object(
+        requests, "get", return_value=ResponseMock(ok=True, json_data=json_data)
+    )
+
+    message = telegram_bot_command["message"]
+    telegram_receiver.handle_bot_command(message=message)
+    assert caplog.messages[0] == "Failed sending message"
+
+
+def test_get_channel_id_from_message(telegram_receiver, telegram_message):
+    message = telegram_message["message"]
+    channel_id = telegram_receiver.get_channel_id_from_message(message=message)
+
+    assert channel_id == str(message["chat"]["id"])
+
+
+def test_get_message_id_from_message(telegram_receiver, telegram_message):
+    message = telegram_message["message"]
+    message_id = telegram_receiver.get_message_id_from_message(message=message)
+
+    assert message_id == str(message["message_id"])
+
+
+def test_get_message_id_from_media_message(telegram_receiver, telegram_media_message_1):
+    message = telegram_media_message_1["message"]
+    message_id = telegram_receiver.get_message_id_from_message(message=message)
+
+    assert message_id == str(message["media_group_id"])
+
+
+def test_get_message_text(telegram_receiver, telegram_message):
+    message = telegram_message["message"]
+    message_text = telegram_receiver.get_message_text(message=message)
+
+    assert message_text == {"text": message["text"], "entities": []}
+
+
+def test_get_message_text_image_message(telegram_receiver, telegram_image_message):
+    message = telegram_image_message["message"]
+    message_text = telegram_receiver.get_message_text(message=message)
+
+    assert message_text == {"text": message["caption"], "entities": []}
+
+
+def test_get_telegram_message_text_with_entities(
+    telegram_receiver, telegram_message_with_entities
+):
+    message = telegram_message_with_entities["message"]
+    message_text = telegram_receiver.get_message_text(message=message)
+
+    assert message_text == {"text": message["text"], "entities": message["entities"]}
+
+
+def test_process_text(telegram_receiver, telegram_message_with_entities):
+    live_post_block = construct_live_post_block(
+        message_id="1234", created=datetime.now()
+    )
+    text = telegram_receiver.get_message_text(
+        message=telegram_message_with_entities["message"]
+    )
+    telegram_receiver.process_text(live_post=live_post_block, message_text=text)
+
+    content = live_post_block["content"]
+    assert content[0].value == "This post contains different type of entities."
+    assert (
+        content[1].value
+        == "This is a regular link <a href='https://github.com/'>https://github.com/</a>."
```

Comment:
Because of quote change proposal this would become: 
```python
        == 'This is a regular link <a href="https://github.com/">https://github.com/</a>.'
```